ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -180,6 +180,20 @@
          (acl/verify-subscription-management-permission
            context :update :provider-object provider-id))))))
 
+(defn check-valid-user
+  "Raise error if the user provided does not exist"
+  [context user-id]
+  (when (or (nil? user-id)
+                (= "" user-id))
```

Comment:
Please fix formatting here, and a safer check may also be  `(string/blank? user-id)` since that will catch nil and whitespace cases